changelog/0.33.md

```diff
@@ -0,0 +1,77 @@
+# v0.33.0
+
+Released 2023-09-28
+
+Known issue:
+
+Added network policies for rook-ceph-crashcollector, fixed in v0.33.1
+The update-ips script will not add the swift IPs in network polices if .objectStorage.sync.syncDefaultBuckets: true, fixed in v0.33.1
+Ingress-nginx controller to 1.8.4 and chart to 4.7.3 (HTTP/2 fix for CVE-2023-44487), fixed in v0.33.1
+a limit of no more than 2 * max_concurrent_streams new streams per one event loop iteration was introduced
+refused streams are now limited to maximum of max_concurrent_streams and 100
```

Comment:
```suggestion
Known issues:

- Missing network policies for rook-ceph-crashcollector, fixed in v0.33.1
- The update-ips script will not add the swift IPs in network polices if .objectStorage.sync.syncDefaultBuckets: true, fixed in v0.33.1
- Ingress-nginx controller to 1.8.4 and chart to 4.7.3 (HTTP/2 fix for CVE-2023-44487), fixed in v0.33.1
  a limit of no more than 2 * max_concurrent_streams new streams per one event loop iteration was introduced
  refused streams are now limited to maximum of max_concurrent_streams and 100
```

Or add a blank line between each entry.

Maybe this should also marked with warning or important.

Were these rendered or did you write them manually?

Reply:
Straight out of https://github.com/elastisys/compliantkubernetes-apps/releases/tag/v0.33.0

Reply:
Well then it's just missing some formatting :sweat_smile: